data/language/en-GB.txt

```diff
@@ -181,6 +182,7 @@ STR_0603    :A wooden style roller coaster with a steel track, allowing for stee
 STR_0604    :Riders ride single file on a narrow monorail track, as they race through tight inversions and direction changes
 STR_0605    :Riders toboggan down a meandering steel track, braking to control their speed
 STR_0606    :An older-style wooden roller coaster with a fast and rough ride, with plenty of air-time, some lateral G’s, and designed to feel ‘out-of-control’
+STR_0607    :A smooth, compact roller coaster on a steel track, where riders ride in spinning, individual cars
```

Comment:
This seems more like a description for a vehicle rather than a track/ride type.

Reply:
I will change this